client/app/components/onboardingGuide/GuideStatusPage.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import scss from './styles.scss';
+
+const GuideStatusPage = (props) => {
+
+  const handleClick = function(e, path) {
+    e.preventDefault();
+    props.changePage(path);
+  }
+
+  const onboarding_data = props.onboarding_data;
+  // TODO: interpolating translation strings needs more thinking
+  const title = props.nextStep ? props.t('title').replace(/\%\{(\w+)\}/g, props.name) : props.t('title_done');
+
+
+  const todo_desc_partial =
+    <div>
+      <p className={ scss.description } >
+        { props.t('description_p1') }
+      </p>
+      <p className={ scss.description } >
+        { props.t('description_p2') }
+      </p>
+    </div>
+
+  const done_desc_partial =
+    <p className={ scss.description }
+      dangerouslySetInnerHTML={{ __html: props.t('congratulation') }} ></p>
```

Comment:
It's due our translation problems. We need to think through how we are going to make interpolation. This project uses translations that are not coming from WTI, which gives us total control over writing those files. So, I added links directly to translation file.
=> translation interpolation problem still exists! It needs to be thought through before we can use WTI.